### PR TITLE
<Explore>: Minor documentation update"

### PR DESCRIPTION
**Pull request by w-endre**
**What this PR does / why we need it**:
Resolves a conflict in the documentation. 

**Which issue(s) this PR fixes**:
None.

**Special notes for your reviewer**:
Feel free to re-phrase the sentence.

### CODE REVIEW

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
The "viewers_can_edit" is mentioned in a separate topic and should be linked correctly.

**Comment by daniellee** [comment]
@w-endre to be able to merge this PR, you need to sign the CLA. Let us know if you are having any problems with that.

**Comment by w-endre** [comment]
> @w-endre to be able to merge this PR, you need to sign the CLA. Let us know if you are having any problems with that.

Thank you for the notification. I signed it.

**Review by achatterjee-grafana — APPROVED** [review]

File: docs/sources/explore/_index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you just want to explore your data and do not want to create a dashboard, the
 
 ## Start exploring
 
-In order to access Explore, you must have an editor or an administrator role. Refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}) for more information on what each role has access to.
+In order to access Explore, you must have an editor or an administrator role, unless the [viewers_can_edit option](viewers_can_edit) is enabled.  Refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}) for more information on what each role has access to.
```
Comment (daniellee) [inline docs/sources/explore/_index.md:21]: Tested this locally and the link is incorrect.

```suggestion
In order to access Explore, you must have an editor or an administrator role, unless the [viewers_can_edit option]({{< relref "../administration/configuration/#viewers_can_edit" >}}) is enabled. Refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}) for more information on what each role has access to.
```

**Pull request merged**